public/scripts/components/sub-components/CurrentStory.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Model from '../../models/StoryModel.js';
+
+export default React.createClass({
+	
+	getInitialState: function () {
+	  	
+		return {story: new Model({id: this.props.id})};
+	 },
+
+	componentDidMount: function () {
+
+		this.state.story.fetch();
+		this.state.story.on('change', (updatedStory) => {
+			this.setState({story: updatedStory});
+
+		});	
+	},
+
+	componentWillUnmount: function () {
+		this.state.story.removeEventListener('change', (updatedStory));
```

Comment:
you want to use the backbone event (un)listener found here: http://backbonejs.org/#Events-off